FinancialTransactionsApi/V1/Helpers/FileGenerator.cs

```diff
@@ -0,0 +1,131 @@
+using CsvHelper;
+using CsvHelper.Configuration;
+using FinancialTransactionsApi.V1.Boundary.Response;
+using FinancialTransactionsApi.V1.Domain;
+using NodaMoney;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using WkHtmlToPdfDotNet;
+using WkHtmlToPdfDotNet.Contracts;
+
+namespace FinancialTransactionsApi.V1.Helpers
+{
+    public static class FileGenerator
+    {
+        public static byte[] WritePdfFile(List<Transaction> transactions, string name, string period, IConverter converter)
+        {
+
+            var report = new ExportResponse();
+            var data = new List<ExportTransactionResponse>();
+
+            report.BankAccountNumber = string.Join(",", transactions.Select(x => x.BankAccountNumber).Distinct().ToArray());
```

Comment:
Has there been a confirmation from the business if we should mask this value when exported?

Reply:
Not yet,for now it is not being exported pending confirmation